libkbfs/ops.go

```diff
@@ -993,59 +993,62 @@ func (ro *rekeyOp) String() string {
 	return "rekey"
 }
 
-func (ro *rekeyOp) CheckConflict(renamer ConflictRenamer, mergedOp op,
+func (ro *rekeyOp) checkConflict(renamer ConflictRenamer, mergedOp op,
 	isFile bool) (crAction, error) {
 	return nil, nil
 }
 
-func (ro *rekeyOp) GetDefaultAction(mergedPath path) crAction {
+func (ro *rekeyOp) getDefaultAction(mergedPath path) crAction {
 	return nil
 }
 
-// gcOp is an op that represents garbage-collecting the history of a
+// GCOp is an op that represents garbage-collecting the history of a
 // folder (which may involve unreferencing blocks that previously held
 // operation lists.  It may contain unref blocks before it is added to
 // the metadata ops list.
-type gcOp struct {
+type GCOp struct {
```

Comment:
Yeah, I think eventually we'll need to export all the op types (as we write more and more checkers) so I figured may as well start now.